src/main/scala/com/databricks/labs/overwatch/validation/SnapValidation.scala

```diff
@@ -127,46 +135,76 @@ class SnapValidation(params: SnapValidationParams,
    * @param params
    * @return
    */
-  protected def snapTable(target: PipelineTable): SnapReport = {
```

Comment:
this function had issues since it was not partitioning the target correctly. Now that it's attached to the Pipeline it shouldn't have any issues as the database object writer.